app/models/grant.rb

```diff
@@ -48,8 +48,8 @@ class Grant < ApplicationRecord
 
   has_many   :reviews,              through: :submissions
 
-  has_many   :applicants,           through: :submissions,
-                                    inverse_of: :applied_grants
+  has_many   :submitters,           through: :submissions,
+                                    inverse_of: :submitted_grants
```

Comment:
not sure about this change. the grant has many applicants. seems like that's what's important here.